pkg/controller/secret_controller.go

```diff
@@ -70,22 +96,15 @@ func (m *SecretController) Reconcile(ctx context.Context, request reconcile.Requ
 		klog.V(6).Infof("secret %s has been deleted.", request.Name)
 		return reconcile.Result{}, nil
 	}
+
+	if err := checkAndCleanOrphanSecret(ctx, m.K8sClient, secrets); err != nil {
+		klog.Warningf("check and clean orphan secret %s error: %v", request.Name, err)
```

Comment:
If it is orphan secret and delete it error, should return err here to make it requeue.

Reply:
secret is requeue each reconcile to make init-config  always up-to-date, https://github.com/juicedata/juicefs-csi-driver/blob/42416b1e68ca0a93187674f7b39cd1b8472aa7da/pkg/controller/secret_controller.go#L128-L129

so I don't think need to requeue here

Reply:
If secret do not contain `token`, it will be passed. 

https://github.com/juicedata/juicefs-csi-driver/blob/42416b1e68ca0a93187674f7b39cd1b8472aa7da/pkg/controller/secret_controller.go#L73-L76

Reply:
ok